dashboards-observability/public/components/event_analytics/explorer/log_patterns/patterns_table.tsx

```diff
@@ -55,24 +55,33 @@ export function PatternsTable(props: PatternsTableProps) {
               0
             )) *
           100;
-        return <EuiText>{`${round(ratio, 2)}%`}</EuiText>;
+        return <EuiText size="s">{`${round(ratio, 2)}%`}</EuiText>;
+      },
+    },
+    {
+      field: 'anomalyCount',
+      name: 'Anomaly Count',
+      width: '6%',
```

Comment:
Seems like Anomaly label column is being cut off - maybe expand this a bit wider?

Reply:
changed to `Anomalies`